packages/third-parties/src/google/index.tsx

```diff
@@ -1,2 +1,3 @@
 export { default as GoogleMapsEmbed } from './GoogleMapsEmbed'
 export { default as YouTubeEmbed } from './YouTubeEmbed'
+export { default as useGoogleTagManager } from './useGoogleTagManager'
```

Comment:
@janicklas-ralph Do you know if the bundle is being tree-shaken with this approach? For example, I import `useGoogleTagManager` in my app, is `GoogleMapsEmbed` and `YouTubeEmbed` included into the total bundle even if they're not used?